Sources/PackageSigning/SignatureProvider.swift

```diff
@@ -55,6 +55,18 @@ public enum SignatureProvider {
             observabilityScope: observabilityScope
         )
     }
+
+    public static func extractSigningEntity(
+        signature: [UInt8],
+        format: SignatureFormat,
+        verifierConfiguration: VerifierConfiguration
+    ) async throws -> SigningEntity {
```

Comment:
```suggestion
    public static func extractSigningEntity(
        signature: [UInt8],
        format: SignatureFormat
    ) async throws -> SigningEntity {
```

If all we want to do is to "peek" the `SigningEntity` and not care if the signing certificate is valid or not, then `VerifierConfiguration` is not necessary.

Reply:
Discussed offline, we need `VerifierConfiguration`.